internal/storing/storing.go

```diff
@@ -0,0 +1,63 @@
+package storing
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path"
+	"time"
+
+	"github.com/lionslon/go-yapmetrics/internal/storage"
+)
+
+func Restore(s *storage.MemStorage, filePath string) {
+	file, err := os.ReadFile(filePath)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	var data storage.AllMetrics
+	if err := json.Unmarshal(file, &data); err != nil {
+		fmt.Println(err)
+	}
+
+	if len(data.Counter) != 0 {
+		s.UpdateCounterData(data.Counter)
+	}
+	if len(data.Gauge) != 0 {
+		s.UpdateGaugeData(data.Gauge)
+	}
+}
+
+func Store(s *storage.MemStorage, filePath string, storeInterval int) {
```

Comment:
```suggestion
func IntervalDump(s *storage.MemStorage, filePath string, storeInterval int) {
```
хочется как-то видеть более явно что тут будет происходит постоянный дамп с интервалом
Можно назваить `DumpWorker` или назвавть пакет `workers`

Reply:
Изменил название функции на `IntervalDump`